src/components/Proposals/ProposalCard.js

```diff
@@ -93,11 +105,17 @@ function ProposalCard({ vote, onProposalClick }) {
           overflow: hidden;
         `}
       >
-        <strong css="font-weight: bold">#{voteId}: </strong>
-        {Array.isArray(description) ? (
-          <Description path={description} />
+        {emptyScript ? (
```

Comment:
I don't love this boolean check for an empty script, but it was the cleanest way I could think to present the branching logic depending on what data is available.